src/index.html

```diff
@@ -7,8 +7,70 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" 
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="img">
+        <a href="#home"><img src="./images/logo.png" alt="moyo"></a> 
+      </div>
+      
+      <nav class="nav">
+        <ul class="nav-list">
+          <li class="nav-li">
+            <a href="#apple" class="nav__link is-active">
+              Apple
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#samsung" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#smartphones" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#laptops&computers" 
+              class="nav__link" 
+              data-qa="hover">
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#gatgets" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#tablets" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#photo" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav-li">
+            <a href="#video" class="nav__link">
```

Comment:
Use single naming style or nav-item or nav__item (as in BEM)

```suggestion
          <li class="nav__item">
            <a href="#video" class="nav__link">
```